Update sets

manticore-projects opened the pull request:
Fixes #1313

Implement complex update sets:
```sql
CREATE TABLE test (
    a        VARCHAR (1)
    , b      VARCHAR (1)
    , c      VARCHAR (1)
)
;
-- queries were supported
update test
set (a, b) = (select '1', '2');

-- value lists were not supported <-- SOLVED
update test
set (a, b) = values( '1', '2');

-- simple expression lists were not supported <-- SOLVED
update test
set (a, b) = ('1', '2');

-- complex expression lists were not supported <-- SOLVED
update test
set (a, b) = (1, (select 2));

-- big boy updates were not supported <-- SOLVED
update test
set (a, b) = (select '1', '2')
      , c = '3'
;
```

wumpz reviewed (commented):
I merged and could not build it. There were multiple pmd failures. After I solved those multiple checkstyle errors kicked in. Could it be, that the gradle build does not run pmd and checkstyle for each build?

manticore-projects commented:
Totally my bad and I would like to apologize. I rushed it and did not pay attention:

1) PMD was active, but an additional filter for PRIORITY=1 was set (I have removed it)
2) Checkstyle was not included, I have added that
3) The GitHub CI quality assurance did not complain, but indicated success. It looks like only 2 out of the 3 usual test ran.

PR #1318 fixes this issues. I have verified that the Gradle build threw the same exceptions as the Maven build. Actually it spotted 2 more issues in the Test Sources based on using lastest CheckStyle Version. I have corrected these too.

Apologies again for sloppy work.
